### PR TITLE
Release prep for v2.0.0

### PR DESCRIPTION
**Pull request by genebean**
Post-merge I will run the new GH action that does a Gem release.

### CODE REVIEW

File: docs/PROVIDER_API.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+```bash
+bundler gem --test=rspec --no-exe --no-ext vmpooler-provider-spoof
+cd vmpooler-providers-spoof/
+mkdir -p ./lib/vmpooler/providers
+touch ./lib/vmpooler/providers/spoof.rb
+mkdir ./lib/vmpooler-providers-spoof
+touch ./lib/vmpooler-providers-spoof/version.rb
+```
+
+There may be some boilerplate files there were generated, just delete those.
```
Comment (suckatrash) [inline docs/PROVIDER_API.md:45]:
```suggestion
There may be some boilerplate files generated, just delete those.
```

**Review by suckatrash — COMMENTED** [review]
Just found a couple typos but otherwise looks great

File: docs/PROVIDER_API.md (hunk 1 above)
Comment (genebean) [reply docs/PROVIDER_API.md]: Good call, fixed

File: README.md [hunk 2]
```diff
@@ -8,11 +8,15 @@ VMPooler provides configurable 'pools' of instantly-available (pre-provisioned)
...
+### v2.0.0 note
+
+As of version 2.0.0, all providers other than the dummy one are now separate gems. Historically the vSphere provider was included within VMPooler itself. That code has been moved to the [puppetlabs/vmpooler-provider-vsphere](https://github.com/puppetlabs/vmpooler-provider-vsphere) repository and the `vmpooler-provider-vsphere` gem. To migrate from VMPooler 1.x to 2.0 you will need to ensure that `vmpooler-provider-vsphere` is installed along side the `vmpooler` gem. See the  [Provider API](docs/PROVIDER_API.md) docs for more information.
+
 ## Installation
 
 ### Prerequisites
 
-VMPooler is available as a gem. To use the gem run `gem install vmpooler` or add it to your Gemfile and install via bundler.
+VMPooler is available as a gem. To use the gem run `gem install vmpooler` or add it to your Gemfile and install via bundler. You will also need to install any needed providers in the same manor.
```
[the remark this answers is not in the document]
Comment (genebean) [reply README.md]: thanks!

**Review by suckatrash — APPROVED** [review]

**Pull request merged**